FirebaseDynamicLinks/Sources/FIRDynamicLinks.m

```diff
@@ -185,11 +185,10 @@ - (void)configureDynamicLinks:(FIRApp *)app {
                        urlScheme:urlScheme
                     userDefaults:nil];
   } else {
-    error =
-        [FIRApp errorForSubspecConfigurationFailureWithDomain:kFirebaseDurableDeepLinkErrorDomain
-                                                    errorCode:FIRErrorCodeDurableDeepLinkFailed
-                                                      service:@"DynamicLinks"
-                                                       reason:errorDescription];
+    error = [self errorForSubspecConfigurationFailureWithDomain:kFirebaseDurableDeepLinkErrorDomain
+                                                      errorCode:FIRErrorCodeDurableDeepLinkFailed
+                                                        service:@"DynamicLinks"
+                                                         reason:errorDescription];
```

Comment:
Since this call is only used once and it's the same number of lines, I think this could even be inlined.

Reply:
Done. Thanks!